packages/video_player/video_player_avfoundation/CHANGELOG.md

```diff
@@ -1,5 +1,6 @@
-## NEXT
+## 2.6.2
 
+* Fixes audio recorded only with first recording.
```

Comment:
This description doesn't make sense in the context of this package, which does not record anything. Please describe the fix in terms of what is changing generally, not the specific effect it has on a specific test app that readers of the changelog won't have context on.